projects/saas-ui/src/app/shared/models/feature.model.ts

```diff
@@ -1,12 +1,15 @@
+import {AnyObject} from '@project-lib/core/api';
 import {FeatureValues} from './feature-values.model';
 
 export class Features {
+  [x: string]: any;
   id?: string;
   name: string;
   description: string;
   key: string;
   type: 'boolean' | 'number' | 'string' | 'object';
-  defaultValue: any;
+  defaultValue: number;
```

Comment:
why is defaultValue type number 
it is string

Reply:
done usme sb types hai default value mai , number bhi string bhi boolean bhi